src/components/Header/Header.jsx

```diff
@@ -1,41 +1,72 @@
 import { Link, useNavigate } from 'react-router-dom'
 import "./Header.css"
 import { useAuth } from '../../contexts/AuthContext'
+import { useCart } from '../../contexts/CartContext'
+import { useWishlist } from '../../contexts/WishlistContext'
 
 export default function Header() {
-    const {loggedIn, logoutHandler} = useAuth()
     const navigate=useNavigate()
-    console.log(loggedIn)
+    const {loggedIn, logoutHandler} = useAuth()
+    const {wishlistCount} = useWishlist()
+    const {cartCount}=useCart()
 
     function handleLogin(e){
```

Comment:
in ES6 syntaxt it will be like single line code so it gives readability to the code